_includes/templates/forms/plain-writing-feedback-2-form.html

```diff
@@ -0,0 +1,294 @@
+<div class="plain-writing-feedback">
+    <section class="block">
+        <div class="content-l
```

Comment:
From team review:
The step blocks are getting removed from the forms for simplicity.